src/locales/en/messages.json

```diff
@@ -3988,6 +3988,24 @@
   "removeUsersWarning": {
     "message": "Are you sure you want to remove the following users? The process may take a few seconds to complete and cannot be interrupted or canceled."
   },
+  "usersHasBeenRemoved": {
```

Comment:
What's this about?

Reply:
This is left over from a rebase i believe. I'll remove it